.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish docs to Wiki
+
+on:
+  push:
+    paths:
+      - docs/**
+    branches:
+      - main
+      
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
+  USER_NAME: team-nimblehq
+  USER_EMAIL: dev@nimblehq.co
+  OWNER: ${{ github.event.repository.owner.name }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  Publish-Docs-To-Wiki:
```

Comment:
this name will need to be rechecked 🤔

Reply:
Is it because of the name, the format, or both? 😁

Reply:
ref 🙈 
https://github.com/nimblehq/android-templates/blob/15428e5e3019ecb5725d48391d7d45f847be8002/.github/workflows/lint_and_unit_test.yml#L6